### ISSUE TITLE
Perform lookup on custom GTK themes in user's $HOME

### ISSUE THREAD

**Issue by darthlukan** [peppermintos/peppermint-control-center#3]
Currently the control center doesn't list user-defined GTK themes that are not installed system-wide, this can cause some inconsistencies in the user experience.

**Comment by kendalltristan** [comment]
Are you adding GTK Theme support to it? Originally it was put together to only have Xfwm themes as I didn't want any overlapping functionality with LXAppearance. This has been a point of confusion in the past and one reviewer wrote off the entire application as meaningless duplication of functionality with LXAppearance even though there's none at all.

Anyway, it should get Xfwm themes in both /usr/share/themes/ and ~/.themes/.

**Comment by darthlukan** [comment]
I was thinking about it. Why use LXAppearance (yet another app installed) if the control center can easily do it? From my understanding of the LXDE ecosystem, LXAppearance is going to either be deprecated completely or have it's functionality altered for LXQT. If we want to be able to provide a simple way for users to adjust the look of their Peppermint system, it makes sense to include that functionality in the control center and then just remove LXAppearance from the ISO.

That's my opinion on the matter anyways, especially considering how jarring it is to have a theme selected in the control center, but it doesn't get applied to all GTK apps consistently.

**Comment by kendalltristan** [comment]
Yeah, pretty much all of LXDE as it has been known is on the way out. I skipped duplicating any LXAppearance functionality because it works well and has been largely unchanged for ages now. I completely agree that ideally it's functionality should be absorbed into the control center at this point (or soon), but that's not my call anymore.

**Comment by darthlukan** [comment]
Well, unless someone has a conniption fit, I'll absorb what makes sense so we can get LXAppearance out of the equation and then any reviewer that might complain about duplicated functionality will see that we just absorbed it, not duplicated it :stuck_out_tongue_winking_eye: . I mean, did the reviewer complain about LXAppearance when it first came out and otherwise replace GTK-chtheme?

**Comment by kendalltristan** [comment]
Hehehe. Good call.

Anyway, I only mention the reviewer because it serves as a practical example of confusion between GTK and window manager themes. It bears keeping in mind because unless you enforce only using themes that have both GTK and Xfwm components, then there's going to be two theme selection options in some capacity. I suppose it would make pretty good sense to label them differently or something. Lessons learned I guess. I'll be the first to admit that the control center is riddled with things that could be improved upon, but hey, that's why you're here. :)

**Comment by darthlukan** [comment]
Oh no, I totally get it and appreciate it. It's always something to think about.  Hopefully the new UI that @Mark-Peppermint has been working on will be enough to wow any reviewers, and assuming I don't monstrously bork something, it works as well as it looks.

You know me, I do what I can and if I can't, I learn it :smile:
